tools/proofers/preview.js

```diff
@@ -856,7 +856,8 @@ $(function () {
             let smallCapRegex = new RegExp(sc1 + "([^]+?)" + sc2, 'g');
 
             function boxHtml(txt) {
-                return txt.replace(/┌/g, "&lt;").replace(/┐/g, "&gt;");
+                return txt.replace(/┌/g, "&lt;").replace(/┐/g, "&gt;")
```

Comment:
Any other characters we need to worry about? Just making sure our solution can scale.

Reply:
I don't think there are.